src/emulator/module/include/module/bridge_args.h

```diff
@@ -96,6 +96,37 @@ struct ArgLayout {
     }
 };
 
+template <typename R0, typename StackHead, typename... StackTail>
+struct ArgLayout<R0, long long, StackHead, StackTail...> {
```

Comment:
It might be worth changing this long long to a specific 64-bit type. Maybe it's not an issue though if it's 64-bit everywhere.

Reply:
It didn't work with uint64_t so I used long long as was indicated in debugger